src/components/PurchaseOrder/Summary/SummaryView.js

```diff
@@ -9,39 +10,62 @@ import {
   KeyValue,
   Row,
 } from '@folio/stripes/components';
+
+import { WORKFLOW_STATUS } from './FieldWorkflowStatus';
 // import css from './SummaryView.css';
 
-const SummaryView = ({ order }) => (
-  <Row>
-    <Col xs={6}>
-      <KeyValue
-        label={<FormattedMessage id="ui-orders.orderSummary.totalUnits" />}
-        value={get(order, 'total_items')}
-      />
-    </Col>
-    <Col xs={6}>
-      <KeyValue label={<FormattedMessage id="ui-orders.orderSummary.approved" />}>
-        <Checkbox
-          checked={get(order, ['approved'])}
-          disabled
-        />
-      </KeyValue>
-    </Col>
-    <Col xs={6}>
-      <KeyValue
-        label={<FormattedMessage id="ui-orders.orderSummary.totalEstimatedPrice" />}
-        value={get(order, 'total_estimated_price')}
-      />
-    </Col>
-    <Col xs={6}>
-      <KeyValue
-        label={<FormattedMessage id="ui-orders.orderSummary.workflowStatus" />}
-        value={get(order, 'workflow_status')}
-      />
-    </Col>
-
-  </Row>
-);
+const SummaryView = ({ order }) => {
```

Comment:
`order` is not marked as required in propTypes, so maybe it makes sense to specify default value like `order={}`

Reply:
done